intezer_sdk/analyses_history.py

```diff
@@ -0,0 +1,160 @@
+                                file_name: str = None,
+                                limit: int = DEFAULT_LIMIT,
+                                offset: int = DEFAULT_OFFSET
+                                ) -> AnalysesResults:
+    """
+    Query for file analyses history.
+    :param start_date: Date to query from.
+    :param end_date: Date to query until.
+    :param api: Instance of public Intezer API for request server.
+    :param aggregate_view: Should the result be aggregated by latest
+    hash/url/computer.
+    :param sources: Filter the analyses by its source.
+    :param verdicts: Filter by the analysis's verdict
+    :param file_name: Filter by the uploaded file's name
+    :param family_names: Filter by the analysis's malicious family name
+    :param file_hash: Filter by the file's hash, in one of the following
+    formats: SHA256, SHA1 or MD5
+    :param limit: Number of analyses returned by the query.
+    :param offset: Number of analyses to skips the before beginning to
+    return the analyses.
+    :return: File query result from server as Results iterator.
+    """
+    filters = generate_analyses_history_filter(
+        start_date, end_date, aggregate_view, sources, verdicts, limit, offset
+    )
+    if file_hash:
+        filters['hash'] = file_hash
+    if family_names:
+        filters['family_names'] = family_names
+    if file_name:
+        filters['file_name'] = file_name
+    return AnalysesResults('/analyses/history', api or get_global_api(), filters)
+
+
+def query_endpoint_analyses_history(*,
+                                    start_date: int,
+                                    end_date: int,
+                                    api: IntezerApi = None,
+                                    aggregate_view: bool = None,
+                                    sources: List[str] = None,
+                                    verdicts: List[str] = None,
+                                    limit: int = DEFAULT_LIMIT,
+                                    offset: int = DEFAULT_OFFSET
+                                    ) -> AnalysesResults:
+    """
+    Query for endpoint analyses history.
+
+    :param start_date: Date to query from.
+    :param end_date: Date to query until.
+    :param api: Instance of public Intezer API for request server.
+    :param aggregate_view: Should the result be aggregated by latest
+    hash/url/computer.
+    :param sources: Filter the analyses by its source.
+    :param verdicts: Filter by the analysis's verdict
+    :param limit: Number of analyses returned by the query.
+    :param offset: Number of analyses to skips the before beginning to
+    return the analyses.
+    :return: Endpoint query result from server as Results iterator.
+    """
+    filters = generate_analyses_history_filter(
+        start_date, end_date, aggregate_view, sources, verdicts, limit, offset
+    )
+    return AnalysesResults(
+        '/endpoint-analyses/history',
```

Comment:
here as well

Reply:
I've just check the origin code and if you will look at the code in github and not here you will see I changed it.
I dont know why here it is not been show.